tests/runner.py

```diff
@@ -65,7 +66,7 @@
 
 def path_from_root(*pathelems):
   """Construct a path relative to the emscripten root directory."""
-  return os.path.join(__rootpath__, *pathelems)
+  return os.fspath(Path(__rootpath__, *pathelems))
```

Comment:
As far as I know the standard way to convert `pathlib.Path` back to a string path, is with `str`,
```suggestion
  return str(Path(__rootpath__, *pathelems))
```

This is equivalent to `os.fspath` since it will call the `__fspath__` method, which in the case of `pathlib.Path` [returns str(self)](https://github.com/python/cpython/blob/f38296feaa3b4d6f215088b21fd2452a4f5c2e4c/Lib/pathlib.py#L743).

The current code is also fine, `str` is just a bit less verbose.

Reply:
done. thanks!